form_designer/admin.py

```diff
@@ -14,6 +14,11 @@
 from form_designer import models
 from xlsxdocument import XLSXDocument
 
+from .models import FIELD_TYPES
+
+
+FIELD_TYPE_CHOICES = ((type["type"], type["verbose_name"]) for type in FIELD_TYPES)
```

Comment:
```suggestion
FIELD_TYPE_CHOICES = ((type["type"], type["verbose_name"]) for type in models.FIELD_TYPES)
```

models are already available in the file, no need to import them again.